docs-ref-services/advisor.md

```diff
@@ -28,28 +28,16 @@ With Advisor, you can:
 Install the Azure Advisor npm module
 
 ```bash
-npm install azure-arm-advisor
+npm install @azure/arm-advisor
 ```
 
 ### Example
 

```

Comment:
I think there should be just one section for samples rather than two (Example and Samples). This made sense when there was an actual example here but now I would just combine the two. As a user I wouldn't know the difference. 

## Samples 
- Examples for using this module in Node.js as well as browser applications can be found in the [README for the Package](https://www.npmjs.com/package/@azure/arm-advisor).
- For more code samples that use various Azure packages, explore the [JavaScript samples](https://docs.microsoft.com/en-us/samples/browse/?languages=javascript).